examples/bigcommerce/README.md

```diff
@@ -125,6 +77,34 @@ BIGCOMMERCE_ACCESS_TOKEN=5lw9ulikcp186tjgg3rs39kh4fg3vci
 
       Look for the ellipsis menu in the left toolbar again and drop the Product price, Product name, and Add to cart button into Makeswift.
 
+## Using your own BigCommerce store
+
+   Here is [a guide](https://support.bigcommerce.com/s/article/Starting-a-BigCommerce-Trial) on how to get your own BigCommerce store going.
+
+   Once you have that, you'll need the environment variables.
+
+   `BIGCOMMERCE_STORE_NAME` and `BIGCOMMERCE_STORE_HASH` can be found in the BigCommerce dashboard
+
```

Comment:
I would add a similar note as before to right here concerning the BIGCOMMERCE_ACCESS_TOKEN